docs/migrate-from-sidecar-to-ambient/migration.adoc

```diff
@@ -0,0 +1,1607 @@
+2. **Ztunnel-level policies will see all traffic as coming from the waypoint**
+3. **Client identity validation should be configured in waypoint-targeted policies**
+
+Ensure your authorization policies account for this identity behavior when validating security enforcement.
+
+**Important**: Enable ALL required waypoints before removing ANY sidecars to maintain policy enforcement.
+
+**If rollback needed**: See <<rollback-procedures,Section 5.2 Rollback Procedures>> for Step 7 waypoint activation rollback instructions (High Risk).
+
+[[step-8-policy-simplification]]
+=== Step 8: Policy Simplification
+
+[[step-81-remove-redundant-sidecar-policies]]
+==== Step 8.1 Remove Redundant Sidecar Policies
+
+Once traffic flows through waypoints, remove duplicate sidecar-specific policies:
+
+[source,console]
+----
+# List all authorization policies to identify duplicates
+kubectl get authorizationpolicy -A
+# List virtual services to identify those replaced by HTTPRoutes
+kubectl get virtualservice -A
+----
+
+Delete the redundant sidecar policies by running the following commands:
+[source,console]
+----
+# Example: Remove sidecar-specific authorization policies
+kubectl delete authorizationpolicy <authorizationpolicy-name> -n <NAMESPACE>
+
+# Example: Remove sidecar-specific virtual services
+kubectl delete virtualservice <virtualservice-name> -n <NAMESPACE>
+----
+
+**If rollback needed**: See <<rollback-procedures,Section 5.2 Rollback Procedures>> for Step 8 policy cleanup rollback instructions (Medium Risk).
+
+[[step-9-remove-sidecar-injection]]
+=== Step 9: Remove Sidecar Injection
+
+[[step-91-disable-sidecar-injection]]
+==== Step 9.1 Disable Sidecar Injection
+
+**Final step**: Remove sidecar injection now that ambient mode is fully operational:
+
+[source,console]
+----
+# Remove sidecar injection while preserving ambient mode labels
+kubectl label namespace bookinfo istio.io/rev- istio.io/dataplane-mode=ambient
+
+# Restart deployments to remove sidecars
+kubectl rollout restart deployment -n bookinfo
+----
+
+[[step-92-validate-sidecar-removal]]
+==== Step 9.2 Validate Sidecar Removal
+
+Verify sidecars are removed and ambient mode is working:
+
+[source,console]
+----
+# Verify pods no longer have sidecars containers in the application pods
+kubectl get pods -n bookinfo
+----
+
+Checking on Kiali should show traffic flowing through ambient mode. You should see traffic flowing through ztunnel and waypoints instead of sidecars.
+
+[[step-93-validate-l7-policy-enforcement]]
+==== Step 9.3 Validate L7 Policy Enforcement
+
+Test that L7 policies are now enforced through waypoints (L7 policies should work as before after waypoint activation):
+
+[source,console]
+----
+# Test authorization policies work through waypoint
+kubectl exec -n bookinfo $(kubectl get pods -n bookinfo -l app=reviews -o jsonpath='{.items[0].metadata.name}') -- curl -v http://reviews.bookinfo:9080/reviews/1
```

Comment:
Looks like we are using the reviews pod to access the reviews service. Is this intentional?